core/jazz_usermanagement/index.js

```diff
@@ -50,7 +51,7 @@ module.exports.handler = (event, context, cb) => {
 		logger.info(JSON.stringify(event));
 
 		if (!event || !event.method || !event.resourcePath) {
-			return cb(JSON.stringify(errorHandler.throwInputValidationError("101", "invalid or missing arguments")));
+			return cb(JSON.stringify(errorHandler.throwInputValidationError("101", "Service operation not supported")));
```

Comment:
Since it is throwing inputValidationError, the existing message is perfect, I believe